go.mod

```diff
@@ -202,7 +210,7 @@ require (
 	go.uber.org/atomic v1.11.0
 	go.uber.org/automaxprocs v1.5.3
 	go.uber.org/dig v1.17.0
-	go.uber.org/fx v1.18.2
+	go.uber.org/fx v1.18.0
```

Comment:
Can you use 1.18.2 to keep the exact same version as before and avoid a downgrade ?